packages/compass/src/app/components/home.tsx

```diff
@@ -8,22 +8,15 @@ import {
   getScrollbarStyles,
   palette,
   resetGlobalCSS,
-  useEffectOnChange,
 } from '@mongodb-js/compass-components';
-import CompassConnections, {
-  SingleConnectionForm,
```

Comment:
I should probably remove the single connection code in a separate PR. Was just temporarily doing it here to be able to see more easily what's going on.